go/private/actions/library.bzl

```diff
@@ -39,8 +40,8 @@ def emit_library(ctx, go_toolchain,
   dep_runfiles = [d.data_runfiles for d in deps]
   direct = depset()
   direct_archives = depset()
-  gc_goopts = tuple(ctx.attr.gc_goopts)
-  cover_vars = ()
+  gc_goopts = [] + ctx.attr.gc_goopts
```

Comment:
`list(ctx.attr.gc_goopts)` would be a little more clear.

Reply:
I am trying to avoid it because it can also convert types (depset to list for instance) 
This cl tries hard to remove all possible depset to list conversions, and not leave anything behind that could easily become one.

Reply:
Got it. I was thinking list + depset was list, but it reports an error.